virttest/libvirt_xml/vm_xml.py

```diff
@@ -1783,29 +1777,171 @@ def __init__(self, virsh_instance=base.virsh):
+
+            __slots__ = ('level', 'associativity', 'policy',
+                         'size_value', 'size_unit', 'line_value', 'line_unit')
+
+            def __init__(self, virsh_instance=base.virsh):
+                """
+                Create new CellCacheXML instance
+                """
+                accessors.XMLAttribute(property_name="level",
+                                       libvirtxml=self,
+                                       forbidden=[],
+                                       parent_xpath='/',
+                                       tag_name='cache',
+                                       attribute='level')
+                accessors.XMLAttribute(property_name="associativity",
+                                       libvirtxml=self,
+                                       forbidden=[],
+                                       parent_xpath='/',
+                                       tag_name='cache',
+                                       attribute='associativity')
+                accessors.XMLAttribute(property_name="policy",
+                                       libvirtxml=self,
+                                       forbidden=[],
+                                       parent_xpath='/',
+                                       tag_name='cache',
+                                       attribute='policy')
+                accessors.XMLAttribute(property_name="size_value",
+                                       libvirtxml=self,
+                                       forbidden=[],
+                                       parent_xpath='/',
+                                       tag_name='size',
+                                       attribute='value')
+                accessors.XMLAttribute(property_name="size_unit",
+                                       libvirtxml=self,
+                                       forbidden=[],
+                                       parent_xpath='/',
+                                       tag_name='size',
+                                       attribute='unit')
+                accessors.XMLAttribute(property_name="line_value",
+                                       libvirtxml=self,
+                                       forbidden=[],
+                                       parent_xpath='/',
+                                       tag_name='line',
+                                       attribute='value')
+                accessors.XMLAttribute(property_name="line_unit",
+                                       libvirtxml=self,
+                                       forbidden=[],
+                                       parent_xpath='/',
+                                       tag_name='line',
+                                       attribute='unit')
+                super(VMCPUXML.CellXML.CellCacheXML, self).__init__(
+                    virsh_instance=virsh_instance)
+                self.xml = '<cache/>'
+
+        def get_cellcache(self):
```

Comment:
maybe need add doc string to explain what the function does here?

Reply:
Added